### DISCUSSION TITLE
Speech Not Clear

### DISCUSSION THREAD

**Discussion by MMingabc** [fishaudio/Bert-VITS2#339] [Q&A]
I am using my own dataset which is a Chinese professional tts speech dataset of 8k utterances to train bert-vits2. The generated speech is not clear. It sounds like the speaker but the content cannot be understood. It's just mumbling. What could be the problem? Thank you. And I am using 16k sampling rate.

**Comment by AngelGuevara7** [comment]
Same problem here for spanish! The alignment matrix looks good but the audio it's just mumbling. I'm using 44,1k sampling rate.
Did you solve it?

**Reply by starmoon-1134** [reply]
This maype help you.
https://github.com/fishaudio/Bert-VITS2/pull/245#issuecomment-1869998937

**Reply by AngelGuevara7** [reply]
Thanks! I will take a look at it.
